python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -27,11 +27,13 @@
 import lsst.geom as geom
 import lsst.meas.algorithms as measAlg
 import lsst.pex.config as pexConfig
+from lsst.pex.exceptions import InvalidParameterError
```

Comment:
I don't think this repo uses `isort`, but the imports appear sorted already and it might be nice to maintain it by moving this down by one line.

Reply:
I'm confused. The imports look sorted to me with this line.

Reply:
Isort sorts imports before from x import